apis/python/src/tiledbsc/util.py

```diff
@@ -66,14 +66,58 @@ def get_sort_and_permutation(lst: list):
 def decategoricalize_array(x):
```

Comment:
we should rename this now that it does more than decategoricalize -- `_to_tiledb_supported_array_type` perhaps